<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Sanity Test"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="0.015">
  <properties />
            <testcase name="verifyUserShouldNavigateToRegisterPagsuccessFully" time="0.015" classname="com.nopcommerce.testsuite.RegisterPageTest">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:70)
  org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:39)
  com.sun.proxy.$Proxy12.toString(Unknown Source)
  com.nopcommerce.pages.HomePage.clickOnRegisterLink(HomePage.java:41)
  com.nopcommerce.testsuite.RegisterPageTest.verifyUserShouldNavigateToRegisterPagsuccessFully(RegisterPageTest.java:24)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
  org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
  org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
  org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
  org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
  org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
  java.util.ArrayList.forEach(ArrayList.java:1249)
  org.testng.TestRunner.privateRun(TestRunner.java:794)
  org.testng.TestRunner.run(TestRunner.java:596)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
  org.testng.SuiteRunner.run(SuiteRunner.java:276)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
  org.testng.TestNG.runSuites(TestNG.java:1063)
  org.testng.TestNG.run(TestNG.java:1031)
  com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
  com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
